### PR TITLE
1.10 update

### PR DESCRIPTION
**Pull request by sschneid**
includes the following component updates:
- Jenkins `2.60.2`
- Marathon `1.5`
- marathon-lb `1.10.0`
- shakedown `1.4.5` (dcoscli `0.5.3`)

### CODE REVIEW

File: conf/cd-demo-app.json [hunk 1]
```diff
@@ -1,28 +1,37 @@
...
-    "upgradeStrategy": {
-        "minimumHealthCapacity": 0
-    },
-    "cmd": "jekyll serve"
+    "portMappings": [
+      {
+        "containerPort": 4000,
+        "hostPort": 0,
+        "protocol": "tcp",
+        "servicePort": 10101,
```
Comment (colin-msphere) [inline conf/cd-demo-app.json]: `servicePort` should probably be left out so the system can choose a random unused port.

File: conf/cd-demo-app.json [hunk 2]
```diff
@@ -1,28 +1,37 @@
...
+        "name": "jenkins-deployed-app"
+      }
+    ]
+  },
+  "cpus": 0.1,
+  "instances": 1,
+  "labels": {
+    "HAPROXY_0_VHOST": "ELB_HOSTNAME",
+    "HAPROXY_GROUP": "external",
+    "lastChangedBy": "velocity-team@mesosphere.com"
```
Comment (colin-msphere) [inline conf/cd-demo-app.json]: This can be removed.

**Review by colin-msphere — CHANGES_REQUESTED** [review]
Two changes for the app definition.

**Comment by sschneid** [comment]
> Two changes for the app definition.

Updated!

**Review by colin-msphere — APPROVED** [review]

**Pull request merged**